packages/core/src/cache/cache.ts

```diff
@@ -133,6 +133,7 @@ class Cache implements ICache {
     }
 
     if (volume.imageData) {
+      volume.imageData.delete();
```

Comment:
shouldn't we delete after decache? decache writes each volume in multiple single slices images